### DISCUSSION TITLE
Instant query - graph - time range issue

### DISCUSSION THREAD

**Discussion by FengZh61** [ccfos/nightingale#1275] [Q&A]
![image](https://user-images.githubusercontent.com/98309003/202404135-fa234d50-2c25-46c1-b708-47cac1bbfd61.png)

我的数据只有最近半个小时的，当我选1小时、3小时，查询结果还是最开始有数据的时间点，夜莺本身就是这么设计的吗？

**Comment by jsers** [comment]
贴下版本信息

**Reply by FengZh61** [reply]
> 贴下版本信息

![image](https://user-images.githubusercontent.com/98309003/202593086-e968767c-91a1-463a-98fa-5f7c6c410ae8.png)

**Comment by UlricQin** [comment]
我看了一下即时查询页面确实是这样的，监控大盘没这个问题 @jsers
